assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -217,13 +217,15 @@ export default function SetupBanner( { onSubmitSuccess } ) {
 
 	if ( variant === null ) {
 		return (
-			<Grid>
-				<Row>
-					<Cell size={ 12 }>
-						<ProgressBar />
-					</Cell>
-				</Row>
-			</Grid>
+			<div className="googlesitekit-publisher-win">
```

Comment:
Hi @sashadoes, this is a good solution, however I'd suggest adding a comment to explain it to help understand why the class is there... Something like this -

> Wrap in the `googlesitekit-publisher-win` class to ensure the output is treated in the same way as `BannerNotification`, with only one instance visible on the screen at a time.